docs/networking.md

```diff
@@ -13,18 +24,29 @@ Use one `--net` command-line argument from cloud-hypervisor to specify the emula
     --disk path=my-root-disk.img \
     --kernel my-vmlinux.bin \
     --cmdline "console=ttyS0 reboot=k panic=1 nomodules root=/dev/vda3" \
-    --net tap=ich0,mac=a4:a1:c2:00:00:01,ip=192.168.4.2,mask=255.255.255.0 \
-          tap=ich1,mac=a4:a1:c2:00:00:02,ip=10.0.1.2,mask=255.255.255.0
+    --net tap=ich0,mac=a4:a1:c2:00:00:01,ip=192.168.4.2,mask=255.255.255.0,num_queues=2,queue_size=256 \
+          tap=ich1,mac=a4:a1:c2:00:00:02,ip=10.0.1.2,mask=255.255.255.0,num_queues=2,queue_size=256
 ```
- 
+
 The `--net` argument takes 1 or more space-separated strings of key value pairs containing the following 4 keys or fields:
 
-| Name     | Purpose                    | Optional  |
-| -------- |----------------------------| ----------|
-| tap      | tap device name            | Yes       |
-| mac      | vNIC mac address           | Yes       |
-| ip       | tap IP IP address          | yes       |
-| mask     | tap IP netmask             | Yes       |
+| Name       | Purpose                    | Optional  |
+| -----------|----------------------------| ----------|
+| tap        | tap device name            | Yes       |
+| mac        | vNIC mac address           | Yes       |
+| ip         | tap IP IP address          | yes       |
+| mask       | tap IP netmask             | Yes       |
+| num_queues | the number of queues       | yes       |
```

Comment:
I like the idea of replacing "num_queues" with "queue_pairs" which would be a pair of rx/tx queues. That would simplify the validation as you do not need to check for an even number and also simplify the documentation.

Reply:
Do you mean to change num_queues from command line? hmm, while, for the original implementation for mq support, I use parameter like queue_pairs, at very early stage, @sboeuf or @sameo would like to change it with num_queues, I can not remember clearly the reason, @sboeuf how do you think about this idea now?

Reply:
It's fine to stick with num_queues, it would be consistent with block then.